### DISCUSSION TITLE
How to run the python code I clone

### DISCUSSION THREAD

**Discussion by candrewtemenoscom** [apache/superset#14142] [Q&A / Help] [answered]
I had cloned the Superset codebase on my local. I am new to Python. Could you please let me know what IDE that I should use and which file to open? I am coming from Visual Studio and Microsoft stack. Thanks so much for your help.

**Comment by etr2460** [comment, marked as answer]
Many developers on Superset use VS Code for their IDE, but any IDE will work for Superset development. You can find instructions for setting up your dev environment in https://github.com/apache/superset/blob/master/CONTRIBUTING.md. Note that we don't officially support Windows for development, but some contributors have made updates to the codebase that theoretically should make Windows development possible.
